rest-api/recipe/models/recipe-models.js

```diff
@@ -208,6 +210,7 @@ async function addRecipeTransaction(body) {
           const new_recipe_ingredients = ingredients.map(
             (ingredient, index) => {
               return {
+                // check this
```

Comment:
Yeah... this seems wrong. We don't want each entry in the `new_recipe_ingredients` array to contain all of the ingredients in the req body, surely? :/

Reply:
I will have a look at it today

Reply:
Done for Fixed

Reply:
Sweet!